rel-eng/tito.props

```diff
@@ -529,3 +529,8 @@ whitelist = katello
   rubygem-runcible
   rubygem-typhoeus
   rubygem-zest
+
+[katello-nightly-pulpcore-el7]
```

Comment:
If we look at the foreman tags then I think `katello-pulpcore-nightly-el7` might be more consistent.

Reply:
maybe, but that's how @pcreech created it in Koji via https://github.com/theforeman/tool_belt/pull/222

Reply:
That's a fair thought, should be easy to edit the tag name I believe. I agree it meets with more of a convention of:

<name>-<version>-<dist>

We should write that convention down.

Reply:
Edited your post to add code tags so Github doesn't attempt to parse it.